.eslintrc.cjs

```diff
@@ -21,6 +21,7 @@ module.exports = {
   extends: [
     'plugin:prettier/recommended', // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
     'plugin:import/recommended',
+    'plugin:unicorn/recommended',
```

Comment:
**nitpick:** according to the comment and the eslint prettier documentation, other plugins should be declared before prettier in the array